app/Http/Controllers/Tests/DestroyControllerUnitTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use App\Actions\Todo\DestroyTodoAction;
+use App\Models\Todo;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Tests\UnitTestCase;
+use function Pest\Laravel\delete;
+
+uses(UnitTestCase::class)
+    ->group('unit', 'controller', 'todo', 'destroy');
+
+beforeEach(function () {
+    $this->action = Mockery::mock(DestroyTodoAction::class);
+    $this->app->instance(DestroyTodoAction::class, $this->action);
+});
+
+
+test('DESTROY /todos/{id}: 200', function () {
+
+    $data = Todo::factory()
+        ->withID(1)
+        ->make();
+
+    $this->action->expects('execute')
+        ->withAnyArgs()
```

Comment:
withAnyArgs it's a bad way. You need to check that the data from the request has gone into the action. Correct in all tests